sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample12_GetSettingsUsingTags.md

```diff
@@ -0,0 +1,54 @@
+# Get Settings With Tags
```

Comment:
I wonder if we should just add these snippets into [Sample2_HelloWorldExtended](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample2_HelloWorldExtended.md) which seems to have similar concepts.

Reply:
I considered doing that but that sample seems to be focused on creating, updating, searching, and deleting a setting with labels. To avoid mixing the two up, I created the new sample. If we want to combine the two, I can add a new section `Search by tags filter` to [Sample2_HelloWorldExtended](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample2_HelloWorldExtended.md) and update one of the settings to include tags.

Reply:
I don't have a strong opinion.  I probably would extend sample 2, but I can't justify throwing away the work that was already done here.    

I leave it to your discretion, but I'm good just going with what we have.